src/Pages/PokemonPage.jsx

```diff
@@ -30,14 +31,127 @@ export const PokemonPage = () => {
     <div>
       {pokemon ? (
         <div className="pokemon-page">
-          <PokemonAvatar pm={pokemon} />
-          <PokemonInfoView pm={pokemon} />
-          <div className="pokemon-move-list">MOVE LIST</div>
-          <div className="pokemon-evo-chain">EVO CHAIN</div>
+          <div className="pm-details">
+              <PokemonAvatar pm={pokemon} />
+              <PokemonInfoBasic pm={pokemon}/>
+              {/* <PokemonInfoView pm={pokemon} /> */}
+          </div>
+          <div className="pm-details2">
+            <div className="info-left">
+                  <PokemonBaseStats pm={pokemon}/>
+                  <PokemonTrainBreed pm={pokemon}/>
+                  <div className="pokemon-evo-chain">EVO CHAIN</div>
+            </div>
+            <div className="pokemon-move-list">MOVE LIST</div>
+          </div>
         </div>
       ) : (
         <LoadingPage />
       )}
     </div>
   );
 };
+
+const PokemonInfoBasic = ({ pm }) => {
+  return (
+    <div className="pokemon-info-basic">
+      <table>
+        <tr>
+          <td>ID:</td>
+          <td>778</td>
+        </tr>
+        <tr>
+          <td>Weight:</td>
+          <td>0.7kg</td>
+        </tr>
+        <tr>
+          <td>Height:</td>
+          <td>0.2m</td>
+        </tr>
+        <tr>
+          <td>Type:</td>
+          <td>Fairy/Ghost</td>
+        </tr>
+        <tr>
+          <td>Abilities:</td>
+          <td>Disguise</td>
+        </tr>
+        <tr>
+          <td></td>
+          <td>Disguise</td>
+        </tr>
+      </table>
+      <p>Dex Description</p>
+      <p>
+        It wears a rag fashioned into a Pikachu costume in an effort to look
+        less scary. Unfortunately, the costume only makes it creepier.
+      </p>
+    </div>
+  );
+};
+
+const PokemonBaseStats = ({ pm }) => {
+  return (
+    <div className="pokemon-base-stats">
+      <table>
+        <tr>
+          <td>HP:</td>
+          <td>55</td>
+          <td>
+            <div className="HP_pm"></div>
+          </td>
+        </tr>
+        <tr>
+          <td>ATK:</td>
+          <td>90</td>
+          <td>
+            <div className="ATK_pm"></div>
+          </td>
+        </tr>
+        <tr>
+          <td>DEF:</td>
+          <td>80</td>
+          <td>
+            <div className="DEF_pm"></div>
+          </td>
+        </tr>
+        <tr>
+          <td>SpA:</td>
+          <td>50</td>
+          <td>
+            <div className="SPA_pm"></div>
+          </td>
+        </tr>
+        <tr>
+          <td>SpD:</td>
+          <td>105</td>
+          <td>
+            <div className="SPD_pm"></div>
+          </td>
+
+        </tr>
+        <tr>
+          <td>SPE:</td>
+          <td>96</td>
+          <td>
+            <div className="SPE_pm"
+            // 米花帮我看看为什么没生效？
+            // 希望达成：div宽度=（数值/255）%
+            style={{width:"(96/255)*100%"}}></div>
```

Comment:
可能是这个？
``{{width: `(96*100/255)`}}``，
我merge之后这边改一下试试

Reply:
我看了一下，应该是用这样的syntax
```suggestion
            style={{width:`${96*100/255}%`}}></div>
```

For reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals